doc/api-docs/scripting.rst

```diff
@@ -260,6 +263,44 @@ for many more complex logging features.  For more
 advanced use of the python logging module, refer to the Python project's
 `Logging Cookbook <https://docs.python.org/2/howto/logging-cookbook.html>`_
 
+=====================================
+Example: Enabling Parallel Processing
+=====================================
+
+.. note::
+   This is an in-development feature and should be used with caution.
+
+Most InVEST models accept an optional entry in the ``args`` dictionary
+representing the number of parallel workers.  Acceptable values for this
+number are:
+
+* ``-1``, representing synchronous execution (this is the default across
+  InVEST)
+* ``0`` representing threaded task management
+* Any other positive integer represents the number of processes that will be
+  created to handle tasks.  ``2*multiprocessing.cpu_count()`` is usually a good
+  number.
+
+.. warning::
+   If you use this feature, you **must** wrap your script in a
+   ``if __name__ == '__main__':`` condition.  Failure to do so will result
+   in a fork bomb (https://en.wikipedia.org/wiki/Fork_bomb).
```

Comment:
Hah, I didn't know there was a term for it!

Reply:
I didn't know about fork bombs either!